_issues/rapist.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Didn't she get a rapist acquitted, and then laugh about the girl who was raped?"
+categories: controversy
+---
+
+**Q:** Didn't Hillary get a rapist acquitted, and then laugh about the girl who was raped?  
+
+**A**: In the second Presidential debate, Donald Trump claimed "Her client she represented
+got him off and she is seen laughing on two occasions laughing at the girl who was raped." This is [completely false](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt). Here are the facts:
+
+* In 1975, Hillary was **appointed by a judge** to defend a homeless man who was being tried for the rape of a 12 year old girl. 
+* The defendant specifically requested a female attorney<iframe width="560" height="315" src="https://www.youtube.com/embed/jJ03Nw7ohww" frameborder="0" allowfullscreen></iframe>
+* Hillary was a young lawyer, only 27 at the time, and **did not want to take the case**. However, since she was appointed, she was not in a position to refuse.
+* The [Sixth Amendment of the U.S. Constitution](http://constitution.findlaw.com/amendment6.html) guarantees the right to an attorney. As a result, Hillary was **obligated to defend her client to the best of her ability**, which she did.
+* The case resulted in a plea bargain, in which **the defendant was convicted** of a lesser charge.
+
+**Q**: But how could Hillary laugh at a girl who was raped?
+
+**A**: She didn't. This accusation stems from [a recorded interview with an Arkansas journalist](https://www.youtube.com/watch?v=e2f13f2awK4) in the 1980's. In the **six minute recording**, Hillary can be heard chuckling four times. However, as reported [in Politico](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt):
+
+> In the recordings, there are spots where Clinton chuckles — but never about the central thrust of the case.
```

Comment:
Maybe clarify "central thrust" our expound on this quote–I feel people might get caught up here.